lib/bolt/transport/local/shell.rb

```diff
@@ -126,7 +126,8 @@ def execute(command, sudoable: true, **options)
 
           if escalate
             if use_sudo
-              sudo_flags = ["sudo", "-k", "-S", "-u", run_as, "-p", Sudoable.sudo_prompt]
+              sudo_exec = target.options['sudo-executable'] || "sudo"
```

Comment:
```suggestion
              sudo_exec = Shellwords.shellescape(target.options['sudo-executable']) || "sudo"
```

Reply:
`Shellwords.join` seems to already escape each element of the array, so I'm not sure if this is needed?

Reply:
Even better!